vortexasdk/api/cargo_movement.py

```diff
@@ -18,7 +18,7 @@ class RawLocations(BaseModel):
 class CargoPortLoadEvent(BaseModel):
     vessel_id: Optional[ID] = None
     start_timestamp: Optional[ISODate] = None
-    event_type: Optional[Literal["cargo_port_load_event"]] = None
+    event_type: Literal["cargo_port_load_event"]
```

Comment:
This is probably something we may want to add in a release changelog. Let's not forget about this

Reply:
These types are deceiving.
In the Python SDK almost every attribute in every model is `Optional`. This does not reflect reality, instead it was a decision made to ensure that in the unlikely event a breaking change was made in the API, it would not break pipelines created by SDK consumers. 
In this particular instance, this field is being used as a discriminator ([Link](https://docs.pydantic.dev/latest/concepts/unions/#discriminated-unions)) and the new version of Pydantic didn't like that it was optional.
In reality, this key is not nullable, so there is no actual change here or risk to SDK users.